src/home_robot/config/control/noplan_velocity_sim.yaml

```diff
@@ -8,8 +8,19 @@ acc_ang: 0.6  # (accel_m_max - accel_m_default) / wheel_separation_m
 lin_error_tol: 0.01
 ang_error_tol: 0.025
 
+# Error tolerance ratio - scale target error by this to a minimum of the tolerance below
```

Comment:
Are you using these somewhere?

Reply:
yes - they're used in the adaptive controller